### PR TITLE
Tooltip

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- Tooltip component created.
- Created left, right, top and bottom tooltip's.
Screen Shot 2022-02-18 at 5 00 46 PM
Screen Shot 2022-02-18 at 5 00 33 PM
Screen Shot 2022-02-18 at 5 01 08 PM
Screen Shot 2022-02-18 at 5 01 22 PM

### CODE REVIEW

**Comment by BALAJI141031** [comment]
looking good but is that border radius(px) is intentional or not?

**Review by SinchanaMS — APPROVED** [review]
Everything looks good!

File: css/tooltip.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.tooltip .tooltip-text-top {
+  bottom: 100%;
+  left: 50%;
+  margin-left: -4.2rem;
+}
+
+.tooltip .tooltip-text-bottom {
+  top: 100%;
+  left: 50%;
+  margin-left: -4.2rem;
```
Comment (m26unkwn) [inline css/tooltip.css:41]: If it is possible you can avoid this negative value

**Review by m26unkwn — APPROVED** [review]
Love this tooltip concept.

**Review by j24m — APPROVED** [review]
10/10 for small PR, class names are good, you have provided with proper indentation everywhere, and the UI looks cute : )

**Pull request merged**